src/Sylius/Behat/Client/ApiClientInterface.php

```diff
@@ -34,7 +34,7 @@ public function delete(string $id): Response;
 
     public function filter(): Response;
 
-    public function sort(string $field, string $value): Response;
+    public function sort(array $sorting): Response;
```

Comment:
Why did you change that?

Reply:
I changed it because i need add next filter for locale, i can add next variable for locale and default add null but in ApiClient I would have to check null and add few lines for version with and without locale.

<img width="628" alt="Screenshot 2020-03-31 at 20 46 20" src="https://user-images.githubusercontent.com/39232096/78063616-ce30a200-7390-11ea-9198-aa3a611ba0da.png">